### PR TITLE
#691: Made instructions for creating password file more obvious.

### PR DESCRIPTION
**Pull request by soulprovidr**
### Relevant Issue (if applicable)
https://github.com/s3fs-fuse/s3fs-fuse/issues/691

### Details
Updated language to eliminate confusion surrounding the credentials to use in the password file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,11 +79,11 @@ The default location for the s3fs password file can be created:
 * using a .passwd-s3fs file in the users home directory (i.e. ~/.passwd-s3fs)
 * using the system-wide /etc/passwd-s3fs file
 
-Enter your S3 identity and credential in a file `~/.passwd-s3fs` and set
+Enter your AWS credentials in a file `~/.passwd-s3fs` and set
```
Comment (gaul) [inline README.md]: This wording is OK although s3fs supports non-AWS S3 implementations.  Perhaps just say credentials here and `ACCESS_KEY_ID` and `SECRET_ACCESS_KEY` below?

Comment (soulprovidr) [reply README.md]: That makes sense, I'll make the change.

**Comment by soulprovidr** [comment]
(Commits are squashed now).

**Comment by ggtakec** [comment]
@soulprovidr @gaul Thanks!

**Pull request merged**